articles/integrating-b2c-mpesa-api-using-node-js/index.md

```diff
@@ -0,0 +1,299 @@
+### Integrating B2C Mpesa API using Node.js
+
+Mpesa can be described as a phone-based mobile banking service. Since its inauguration in 2007 by Vodafone Group PLC and Safaricom in Kenya, it has expanded to eight more countries in Africa.
+
+An API (Application Programming Interface) is an intermediary between two software applications that enables them to communicate.
+
+B2C Mpesa API enables software developers to integrate the functionality of merchants sending payments to their customers in their applications.
+
+### Goals
+
+In this article, we will integrate the B2C Mpesa API to a Node.js RESTful API. We will consume the API using [swagger](https://swagger.io/).
+
+### Prerequisites
+
+To follow along in this article, it is helpful to have the following:
+
+- [Node.js](https://nodejs.org/en/) installed on your computer.
+- Some basic knowledge working with JavaScript.
+- A text editor installed.
```

Comment:
Inform the reader what text editor you will be using

Reply:
done.